### DISCUSSION TITLE
Conversations visible for development but not visible for production

### DISCUSSION THREAD

**Discussion by PemariAdmin** [botfront/botfront#955] [Q&A] [answered]
Hi all,

I've not before set up botfront for development along with additional 'production' instances of rasa and actions services.  Now I have I am able to view the 'development' conversations/new utterances but not the 'production' conversations and the link to conversations in the new utterance screen for production does not link to the correct conversation;  the new utterances shown for either development and production look to be correct for the relevant environment.

I am using Botfront 1.0.3/Docker installs.

So my question - should this work?  If yes, any pointers to where I should be looking?

Thanks

**Comment by PemariAdmin** [comment, marked as answer]
Hi, an update. I managed to now get this working.  It was an issue on the browser side with webchat.  By default it is using localStorage and saves the sessionId.  When I changed this Webchat to point at the production rasa having first been using it to talk to dev rasa, the    conversation was a development conversation and not a production conversation so I could see the new incoming utterances but not the conversation.

I 'fixed' it by setting the storage param on the webchat startup to session and now my conversations for production are appearing....  We probably want to use session based anyhow so good for me.  I could also have simply cleared the cache...

**Comment by Ameerak124** [comment]
Hi @andylitton  I want to Productionize botfront chatbot . Im helpless, please help me
